.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Contributing
+
+TODO
```

Comment:
You can take inspiration from https://github.com/github/docs/blob/main/CONTRIBUTING.md

Reply:
Yes, but I left it empty to discuss it with the whole group.